incubator/cloudbeaver/templates/statefulsets.yaml

```diff
@@ -0,0 +1,159 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "cloudbeaver.fullname" . }}
+spec:
+  serviceName: cloudbeaver
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cloudbeaver
+  template:
+    metadata:
+      labels:
+        app: cloudbeaver
+    spec:
+      serviceAccountName: {{ include "cloudbeaver.serviceAccountName" . }}
+      initContainers:
+        - name: source-jars
```

Comment:
Nitpick: Do we need to also set the `workingDir: /opt/cloudbeaver` similar to the second init container?

Reply:
Since this one uses the cloudbeaver image, the working dir is already /opt/cloudbeaver as set in their docker file - but it could be added to overcome any changes that might happen in the cloudbeaver image.

Reply:
Done!